src/commands/topics.test.ts

```diff
@@ -114,22 +127,6 @@ describe("commands/topics.ts produceMessageFromDocument()", function () {
     assert.ok(errorMsg.startsWith("Failed to produce 1 message to topic"), errorMsg);
   });
 
-  it("should show an error notification for any nested error_code>=400 responses", async function () {
-    // response will show status 200, but the error is nested in the response body
-    clientStub.produceRecord.resolves({
-      error_code: 422,
-      message: "uh oh",
-      timestamp: new Date(),
-      partition_id: 0,
-    });
```

Comment:
No longer needed since the sidecar should be floating error status/text to the main response instead of it being nested in the body